test/Microsoft.AspNet.Mvc.DataAnnotations.Test/MvcDataAnnotationsMvcBuilderExtensionsTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.DataAnnotations.Internal;
+using Microsoft.AspNet.Mvc.Localization;
+using Microsoft.AspNet.Mvc.ModelBinding.Validation;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.Localization;
+using Microsoft.Framework.OptionsModel;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.DataAnnotations.Test
+{
+    public class DataAnnotationsLocalizationServiceTest
+    {
+        [Fact]
+        public void AddDataAnnotationsLocalizationServices_AddsNeededServices()
```

Comment:
This seems like the kind of test that would change if you made an insignificant change to the product (lifetime, localization adds another service). Do you think there's a lot of value coming from these?

I wouldn't bother as this seems really fragile.